### DISCUSSION TITLE
Asociate check with PR

### DISCUSSION THREAD

**Discussion by spothound** [community/community#25577] [Actions]
<p>Hello world!</p>
<p>Is there any way to associate a check/action output to a certain PR without using the <code>on: pullrequest</code> or <code>on: push</code> code?</p>
<p>For example, when someone comment on a PR, run an action and create a check associate for such PR.</p>
<p>Best regards.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Pull request is an issue. So you can use <strong>on: issue_comment.</strong> to trigger a workflow when a new comment added to a PR.  </p>
<p>And you can use on:  pull_request_review_comment to run your workflow anytime a comment on a pull request’s unified diff is modified</p>
<p>Please refer to this document: <a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhelp.github.com%2Fen%2Factions%2Fautomating-your-workflow-with-github-actions%2Fevents-that-trigger-workflows%23issue-comment-event-issue_comment&amp;data=02%7C01%7Cv-yjingz%40microsoft.com%7C28594812381449008c7b08d76cc9dc4d%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637097486550573430&amp;sdata=5UxOuKS3KnML3kqGSei96%2F0xe2oGBIuGVUWDXjfD898%3D&amp;reserved=0" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#issue-comment-event-issue_comment</a></p>
<p>I want to note that, you cannot see these types of checks in PP &gt; Checks tab. You need to find it in Repo&gt; Actions &gt; Workflows list…</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/234d5eefa0a50b6ee1c06175f87dab817ab004c0.png" data-download-href="/uploads/short-url/52itCpsRjJqt1YV96AA8edE6ihG.png?dl=1" title="pull request comment.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181091873-09546da3-4816-4daf-98c4-fdd0f3b95376.png" alt="pull request comment.png" data-base62-sha1="52itCpsRjJqt1YV96AA8edE6ihG" width="690" height="89" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/234d5eefa0a50b6ee1c06175f87dab817ab004c0_2_690x89.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/234d5eefa0a50b6ee1c06175f87dab817ab004c0_2_1035x133.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/2/234d5eefa0a50b6ee1c06175f87dab817ab004c0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/234d5eefa0a50b6ee1c06175f87dab817ab004c0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">pull request comment.png</span><span class="informations">1378×178 10.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/f4ec1ec2d15dfa26d87c8b310da444fdf5714445.png" data-download-href="/uploads/short-url/yWGo35lvTuEhhCBMjnturUXYT3v.png?dl=1" title="PR checks.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181091875-a1cc3e58-a6bd-4985-bc7a-0879deae6a97.png" alt="PR checks.png" data-base62-sha1="yWGo35lvTuEhhCBMjnturUXYT3v" width="627" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/f4ec1ec2d15dfa26d87c8b310da444fdf5714445_2_627x500.png, https://aws1.discourse-cdn.com/github/original/2X/f/f4ec1ec2d15dfa26d87c8b310da444fdf5714445.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/f/f4ec1ec2d15dfa26d87c8b310da444fdf5714445.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/f4ec1ec2d15dfa26d87c8b310da444fdf5714445_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">PR checks.png</span><span class="informations">783×624 30.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
